### DISCUSSION TITLE
How to offer translation help?

### DISCUSSION THREAD

**Discussion by PhL38F** [aclap-dev/video-downloadhelper#104] [Help and Q&A] [answered]
### Your question

Hello
The title says it all...
Regards
PhL38Fr

**Comment by jcfrog** [comment, marked as answer]
Hi,
Please have a look here: https://github.com/aclap-dev/video-downloadhelper/discussions/110

**Comment by PhL38F** [comment]
Thank you for adding that!envoyé : 16 février 2024 à 07:17de : jcfrog ***@***.***>à : aclap-dev/video-downloadhelper ***@***.***>cc : PhL38F ***@***.***>, Author ***@***.***>objet : Re: [aclap-dev/video-downloadhelper] How to offer translation help? (Discussion #104) Hi, Please have a look here: #110—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
